tooling/gulpfile.js

```diff
@@ -28,27 +28,26 @@ const rootFolder           = path.join(__dirname, '../');
 
 // General configurations for the build process
 const config = {
-  libName: libName,
+  libName,
   paths: {
-    rootFolder: rootFolder,
+    rootFolder,
     distFolder: path.join(rootFolder, 'dist/terminus-ui'),
     npmFolder: path.join(rootFolder, 'node_modules/@terminus/ui'),
     scss: {
       helpersInputs: [
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_a11y.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_typography.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_cursors.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_color.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_animation.scss'),
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_assets.scss'),
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_breakpoints.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_color.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_cursors.scss'),
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_layout.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_input-placeholder.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_z-index.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_menu.scss'),
-        // Spacing must be after typography
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_spacing.scss'),
-        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_animation.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_scrollbars.scss'),
         path.join(rootFolder, 'terminus-ui', 'scss/helpers/_shadows.scss'),
+        path.join(rootFolder, 'terminus-ui', 'scss/helpers/_spacing.scss'),
```

Comment:
Previously there was a comment _spacing should be after _typography. Did something change, or should this move and the comment be restored?

Reply:
Spacing used to depend on `typography` to know the base font size. We've since duplicated this base size into spacing directly so that we don't have to depend on that import.

End goal we will get all our tokens injected together from another tool to avoid this completely.

Reply:
👍